src/Frontend/config/counterpart.js

```diff
@@ -10,15 +10,11 @@
 
 'use strict';
 
-import { jsdom } from 'jsdom';
+import de from './languages/de';
+import en from './languages/en';
 import counterpart from 'counterpart';
-import de from '../config/languages/de';
-import en from '../config/languages/en';
+import Locale from '../util/http/LocaleService';
 
+Locale.setLocale(null);
```

Comment:
requires a comment as setting the locale to ``null`` may seem a little bit odd during debug